test/clj_gatling/simulation_test.clj

```diff
@@ -406,65 +420,47 @@
       (is (= true (get-result (:requests result) "Request1")))
       (is (= false (get-result (:requests result) "Request2"))))))
 
-(deftest simulation-returns-result-when-run-with-multiple-scenarios-with-one-user
-  (let [result (run-two-scenarios {:name "Test scenario"
-                                   :steps [(step "Step" true)]}
-                                  {:name "Test scenario2"
-                                   :steps [(step "Step" true)]}
-                                  :concurrency 2)]
-    ;;Stop condition is not synced between parallel scenarios
-    ;;so once in a while there might be one extra scenario
-    ;;This is ok tolerance for max requests
-    (is (equal? (first (filter #(= 0 (:id %)) result)) {:name "Test scenario"
-                                                        :id 0
-                                                        :start number?
-                                                        :end number?
-                                                        :requests anything}))
-    (is (equal? (first (filter #(= 1 (:id %)) result)) {:name "Test scenario2"
-                                                        :id 1
-                                                        :start number?
-                                                        :end number?
-                                                        :requests anything}))))
```

Comment:
IMO this test is not valid any more. It's not possible to have multiple scenarios with a 'single user'. There is an assertion that the user count is at least the same or more than the scenario count.

The only reason this ever worked is because with no explicit users, the default is set to `(range concurrency)`, which effectively gives 2 users. The change to the code now requires the number of users to be passed in to be explicit. Which is fine, that will happen anyway for anyone calling `gatling.core/run`, it only caused issues for test functions, which dove into the function sequence part way through.